### ISSUE TITLE
(unwanted) bold font in context menu

### ISSUE THREAD

**Issue by antodc** [doublecmd/doublecmd#1917]
### Version

Double Commander
Version: 1.1.19 gamma
Revision: 585
Commit: 9dd31c2e9
Build date: 2024/10/06
Lazarus: 3.7.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 10 22H2 x86_64

### Description

I have Cygwin installed on my pc and I used the following commands to add two items to the context menu. They open mintty and cmd terminals, respectively:

chere -i -t cmd -s cmd -e "Cmd Prompt Here"
chere -i -t mintty -s zsh -e "Zsh Prompt Here"

For some reason, the first item "Cmd Prompt Here" appears in bold when I right click on an empty space in DC. It appears correctly (i.e. no bold) in Explorer. It also appears correctly in DC when right clicking on a directory.

**Comment by antodc** [comment]
Some additional information.
It seems that the first item (in alphabetic order) added to this registry key
\HKEY_CLASSES_ROOT\Directory\background\shell
is always shown with a bold font in the context menu that pops up when right-clicking on an empty space in DC (but not in Explorer).

**Comment by alexx2000** [comment]
Under Windows 11 the first item is bold only at the first call.

DC creates menu using a system function. So it is a hard to change font of existing menu item.

**Comment by antodc** [comment]
I read somewhere on the web that the bold item is the default command that Windows executes when double-clicking the object. As Windows does nothing when one double-clicks on an empty space there is no bold item in the corresponding context menu. I was wondering whether the behaviour in DC may be related to the possibility that in DC double-clicking the empty area can actually do something. I mean, going up one directory. But I tried to enable/disable that option and nothing changed.
Anyway, sorry to bother you with this minor issue. I will wait an upgrade to Windows 11. In the meantime, I added an empty line to that context menu, so it gets the 'bold' and it does not disturb me. :-)
